lib/ours/util.js

```diff
@@ -1,6 +1,9 @@
 'use strict'
 
 const bufferModule = require('buffer')
+const {
+  codes: { ERR_INVALID_ARG_TYPE }
+} = require('./errors')
```

Comment:
This creates a cyclic dependency between `ours/util.js` and `ours/errors.js`... 😖

I'll try to split up these files to remove cycle.